demo/site/src/common/blocks/RichTextBlock.module.scss

```diff
@@ -0,0 +1,47 @@
+@use "@src/styles/variables" as *;
+
+.disableLastBottomSpacing {
+    > *:last-child {
+        margin-bottom: 0;
+
+        @media (min-width: $breakpoint-sm) {
+            margin-bottom: 0;
+        }
+    }
+}
+
+.text {
+    white-space: pre-line;
+
+    // Show empty lines as spacing between paragraphs
+    &:empty:not(:first-child:last-child)::before {
+        white-space: pre;
+        content: " ";
+    }
+}
+
+.inlineLink {
+    color: var(--palette-primary-main);
+    transition: color 0.3s ease-in-out;
+
+    &:hover {
+        color: var(--palette-primary-dark);
+        text-decoration-thickness: 2px;
+    }
+}
+
+.pageLayoutContent {
+    grid-column: 3 / -3;
+}
+
+.ol-level-0 {
```

Comment:
that selector is now dash-separated instead of camel case.

Reply:
we need rules for that, if its not allowed 🤔

Reply:
No not yet

Reply:
> CSS Modules recommends kebap-case, see https://github.com/css-modules/css-modules/blob/master/docs/naming.md.

You mean camel case 😅